nemo_skills/training/openrlhf/llm_reward.py

```diff
@@ -1,21 +1,40 @@
 import torch
-from nemo_skills.inference.server.model import get_model
-from nemo_skills.prompt.utils import get_prompt
+
 from nemo_skills.code_execution.math_grader import extract_answer
 from nemo_skills.evaluation.metrics.utils import is_correct_judgement
+from nemo_skills.inference.server.model import get_model
+from nemo_skills.prompt.utils import get_prompt
+from nemo_skills.utils import prefill_judgement
 
 
 def reward_func(queries: list[str], prompts: list[str], prompt_metadata: list[dict]):
```

Comment:
I am confused by the parameter `prompts`. It's passed as a parameter, and then the variable `prompts` is created inside the function on line 29. The other variables could also use better names. For e.g., `queries` could be `generations`/`responses`.

Reply:
queries/prompts is the original notation from openrlhf, so probably best to keep it (at least for now). Agree it's a bit confusing, let me add some docs for that